provision.yaml

```diff
@@ -1,31 +1,36 @@
 ---
-
 local:
   provisioner: docker
   images:
-    - 'litmusimage/centos:8'
+    - "litmusimage/centos:8"
 
 linux:
   provisioner: docker
-  images: 
-    - 'litmusimage/centos:7'
-    - 'litmusimage/centos:8'
-    - 'litmusimage/debian:9'
-    - 'litmusimage/debian:10'
-    - 'litmusimage/debian:11'
-    - 'litmusimage/ubuntu:16.04'
-    - 'litmusimage/ubuntu:18.04'
-    - 'litmusimage/ubuntu:20.04'
-    - 'litmusimage/oraclelinux:6'
-    - 'litmusimage/oraclelinux:7'
-    - 'litmusimage/scientificlinux:7'
-    - 'litmusimage/rockylinux:8'
-    - 'litmusimage/almalinux:8'
+  images:
+    - "ghcr.io/ffalor/centos:7"
+    - "ghcr.io/ffalor/centos:stream8"
+    - "ghcr.io/ffalor/centos:stream9"
```

Comment:
@ffalor we actually don't support centos stream at this point in time. So could present problems when running.

Reply:
Yeah, the centos 9 is not in the `package.json` as supported so it won't show up on the forge module. This is more to keep track of what the API is doing. This pipeline is what caught the recent `16/18/20/22` api change for Ubuntu/Debian.

Idc to remove it either.

Reply:
Cool. Got it and makes sense.